### PR TITLE
Corrected spelling mistakes

### PR DESCRIPTION
**Pull request by P45QU10U**

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@P45QU10U thank you for the time and effort you put into this. A couple of things i want to discuss with you about this pull request.
- If you don't mind, can you change the title for the pull request? My french is a bit "rusty" (pardon the bad pun), but understand the meaning. But bear in mind that the rest of the maintainers are not native french speakers/writers and will have a bit of a hard time understanding the pull request. Also if you could provide a clear and concise description it would go a long way.
- Content wise, like i said above, my french is a bit rusty, so i might not be the best person to go about this and merge it and possibly introduce misleadings. As part of the translation process that we have in place, i'm going to reach out the community and see if the changes you're proposing are ok and if so, i will gladly merge it. 

Do you mind waiting a bit?

Stay safe

**Comment by P45QU10U** [comment]
OK for the title. That's better.

File: content/intro-to-storybook/react/fr/get-started.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ yarn start
 ```
 
 <div class="aside"> 
-Vous avez peut-être remarqué que nous avons ajouté le drapeau <code>--watchAll</code> à notre commande de test, ne vous inquiétez pas, c'est intentionnel, ce petit changement permettra de s'assurer que tous les tests sont effectués et que tout va bien avec notre application. Pendant que vous progressez dans ce tutoriel, vous serez exposé à différents scénarios de test. Pour vous assurer que votre suite de tests fonctionneront dans leur intégralité vous pouvez aussi ajouter ce drapeau à votre script de test dans votre <code>package.json</code> .
+Vous avez peut-être remarqué que nous avons ajouté le drapeau <code>--watchAll</code> à notre commande de test, ne vous inquiétez pas, c'est intentionnel, ce petit changement permettra de s'assurer que tous les tests sont effectués et que tout va bien avec notre application. Pendant que vous progressez dans ce tutoriel, vous serez exposés à différents scénarios de test. Pour vous assurer que votre suite de tests fonctionnera dans son intégralité, vous pouvez aussi ajouter ce drapeau à votre script de test dans votre <code>package.json</code> .
```
Comment (Draketheb4dass) [inline content/intro-to-storybook/react/fr/get-started.md:45]: I think "vous serez exposé" is correct. For the rest good job

**Review by Draketheb4dass — CHANGES_REQUESTED** [review]

**Comment by Draketheb4dass** [comment]
Everything is fine. good job @P45QU10U . Except for the suggested change.

File: content/intro-to-storybook/react/fr/get-started.md (hunk 1 above)
Comment (P45QU10U) [reply content/intro-to-storybook/react/fr/get-started.md:45]: I'm sure that it's : vous serez exposés.

Comment (Draketheb4dass) [reply content/intro-to-storybook/react/fr/get-started.md:45]: Ok thanks. Good job. Good to be merged

**Review by Draketheb4dass — APPROVED** [review]

File: content/intro-to-storybook/react/fr/get-started.md (hunk 1 above)
Comment (P45QU10U) [reply content/intro-to-storybook/react/fr/get-started.md:45]: Thank you

**Comment by jonniebigodes** [comment]
@P45QU10U thank you very much for the revision and @Draketheb4dass thank you as well for taking the time to triage this.

Going to merge this pull request.

Stay safe both of you

**Pull request merged**